src/rogue/interfaces/memory/Master.cpp

```diff
@@ -68,7 +68,7 @@ void rim::Master::setup_python() {
 //! Create object
 rim::Master::Master() {
    error_   = "";
-   slave_   = rim::Slave::create(4,4); // Empty placeholder
+   slave_   = rim::Slave::create(4,0); // Empty placeholder
```

Comment:
Why this change?

Reply:
It sets the default Slave placeholder module for a master to have an invalid max size which triggers the error in the Block. Otherwise their is no way to tell if the memory device is not connected.